datastore/integration-test/response/metadata/200/items_get_single_platform.json

```diff
@@ -0,0 +1,1643 @@
+{
```

Comment:
Can we pick a platform with less timeseries? `06225` has only three.

To be honest, I was a bit shocked by that this PR has way more lines then the whole API up to now... :-(